.github/workflows/build-docs.yaml

```diff
@@ -1,56 +1,106 @@
-name: Build and Deploy
-on:   
+name: Build and Deploy Documentation
+
+on:
   schedule:
     - cron: '0 1 * * *'
   workflow_dispatch:
+
 jobs:
-  build-and-deploy-docs:
-    runs-on: ubuntu-latest
-    steps: 
-      - name: Get repository and install Rust dependencies
+  build-docs:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
I'm slightly confused about this matrix.  It seems steps either run on one platform or the other without much shared behavior.  What do you think about splitting this into 2 jobs?

Reply:
Very fair point! Will do 👍